wdl/tasks/awk.wdl

```diff
@@ -62,3 +62,50 @@ task shift_bed_reads_start_and_end_range {
     }
 }
 
+
+task clean_reference_fasta_headers {
+    input {
+        File fasta
+        Resources resources
+        String out = "cleaned.fa" 
+    }
+
+    command <<<
+        awk '/^>/{$0=$1} 1' ~{fasta} \
```

Comment:
Looks good but can we mutliline these like https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/ca6bfbb3172cc490be7fc9bf10bd99ca129579f3/wdl/tasks/awk.wdl#L45

Reply:
sure.